misk-redis/src/main/kotlin/misk/redis/FakeRedis.kt

```diff
@@ -31,6 +53,11 @@ class FakeRedis : Redis {
   }
 
   override fun get(key: String): String? {
```

Comment:
does it even make sense to support this method? if you use a `ByteString`, a client can easily call `utf8()` to do the conversion and it's explicit

Reply:
Yeah I was conflicted with this, the reason I supported it is because storing a string & retrieving a string is a common use case, but I'm not against doing what you said either since it'd simplify the api quite a bit. I'll follow your suggestion.